bi-connectors/PowerBIConnector/docs/OpenSearchProject.md

```diff
@@ -64,3 +64,16 @@
 2. Check if auth credentials are correct.
 3. Check if server is running.
 
+## Limitations and issues

```

Comment:
Limitations and Known Issues

Reply:
Updated in e89be25a0afeb0eb8dbe366e461a19f7144c9b6b